<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlashDataV7</class>
 <widget class="QWidget" name="FlashDataV7">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FlashDataV7</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="cbAdditionalInfo">
     <property name="text">
      <string>Use additional information</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbAdditionalInfo">
     <property name="title">
      <string>Additional information</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Module identifier : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leModuleIdentifier">
          <property name="toolTip">
           <string>16 Byte Hexadecimal</string>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="maxLength">
           <number>32</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Module serial :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leModuleSerial">
          <property name="toolTip">
           <string>19 Byte</string>
          </property>
          <property name="maxLength">
           <number>19</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Module suffix :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leModuleSuffix">
          <property name="toolTip">
           <string>16 Byte</string>
          </property>
          <property name="maxLength">
           <number>16</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cbAdditionalInfo</sender>
   <signal>toggled(bool)</signal>
   <receiver>gbAdditionalInfo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
